stores/toolbarTitle.ts

```diff
@@ -11,5 +11,9 @@ export const toolbarTitleStore = defineStore("toolbarTitle", () => {
     );
   }
 
-  return { setReactiveToolbarTitle, toolbarTitle };
+  function setToolbarTitle(title: string) {
+    toolbarTitle.value = title;
+  }
+
+  return { setReactiveToolbarTitle, toolbarTitle, setToolbarTitle };
```

Comment:
This implementation is prone to the problem, that the title will be overwritten if the reactive title is updated late.